flexisip/Containerfile

```diff
@@ -0,0 +1,16 @@
+FROM docker.io/library/debian:buster-slim as base
+
+FROM base as production
+RUN apt-get update && apt-get install -y wget gnupg2
+RUN echo 'deb [arch=amd64] http://linphone.org/snapshots/debian buster stable' > /etc/apt/sources.list.d/belledonne.list
+RUN wget https://linphone.org/snapshots/debian/keyring.gpg -O - | apt-key add -
+RUN apt-get update && apt-get install -y libmariadb-dev && apt-get install -y bc-flexisip=2.1.5-1
+RUN apt-get -y remove wget gnupg2 && apt -y autoremove && apt-get -y clean
+
+RUN addgroup --system --gid 991 asterisk && adduser --system --uid 990 --shell /bin/false --home /var/lib/asterisk --gecos 'Asterisk User' --ingroup asterisk asterisk
+RUN mkdir -p /etc/flexisip/tls && chown asterisk:asterisk /etc/flexisip
+RUN mkdir -p /etc/asterisk && chown asterisk:asterisk /etc/asterisk
```

Comment:
It's possible to squash all this operations in one layer, I can get to it tomorrow morning

Reply:
It's a matter of having less layers (from 11 down to 4), helps podman by not downloading all the layers (which can add issues during downloads) and a bit of "best practice".

However I've got a better idea, let's squash all the layers, since we're not reusing any of it anyway, this way we won't have to optimize the Containerfile.